### ISSUE TITLE
VerticalBarChart Crash: When I have only one bar

### ISSUE THREAD

**Issue by Wavesonics** [KoalaPlot/koalaplot-core#18]
I'm not sure of the exact cause yet.

```
Exception in thread "AWT-EventQueue-0" java.lang.IndexOutOfBoundsException: Index -1 out of bounds for length 1
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:359)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at io.github.koalaplot.core.bar.VerticalBarChartKt$VerticalBarChart$4.measure-3p2s80s(VerticalBarChart.kt:144)
	at androidx.compose.ui.node.InnerNodeCoordinator.measure-BRTryo0(InnerNodeCoordinator.kt:103)
	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$performMeasure$2.invoke(LayoutNodeLayoutDelegate.kt:1090)
	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$performMeasure$2.invoke(LayoutNodeLayoutDelegate.kt:1086)
	at androidx.compose.runtime.snapshots.Snapshot$Companion.observe(Snapshot.kt:2201)
	at androidx.compose.runtime.snapshots.SnapshotStateObserver$observeReads$1$1.invoke(SnapshotStateObserver.kt:238)
	at androidx.compose.runtime.snapshots.SnapshotStateObserver$observeReads$1$1.invoke(SnapshotStateObserver.kt:234)
	at androidx.compose.runtime.SnapshotStateKt__DerivedStateKt.observeDerivedStateRecalculations(DerivedState.kt:341)
	at androidx.compose.runtime.SnapshotStateKt.observeDerivedStateRecalculations(Unknown Source)
```

**Comment by gsteckman** [comment]
I spotted a bug looking at the line VerticalBarChart.kt:144, that would occur only if the data series has 1 element. Thanks for providing the stack trace.

**Comment by gsteckman** [comment]
Fixed in [8b9f427](https://github.com/KoalaPlot/koalaplot-core/commit/8b9f4275d7c8a6fb5907fbcb85cb885187dff511)
